channels-rke2.yaml

```diff
@@ -1895,3 +1895,26 @@ releases:
         repo: rancher-rke2-charts
         version: v4.0.2-build2024020802
     featureVersions: *featureVersions-v1
+  - version: v1.29.3+rke2r1
+    minChannelServerVersion: v2.9.0-alpha1
+    maxChannelServerVersion: v2.9.99
+    serverArgs: *serverArgs-v1-28-8-rke2r1
+    agentArgs: *agentArgs-v1-28-8-rke2r1
+    charts: &charts-v1-29-3-rke2r1
+      <<: *charts-v1-28-8-rke2r1
+      rancher-vsphere-csi:
+        repo: rancher-rke2-charts
+        version: 3.1.2-rancher400
+      rke2-canal:
+        repo: rancher-rke2-charts
+        version: v3.27.3-build2024042301
+      rke2-calico:
+        repo: rancher-rke2-charts
+        version: v3.27.300
+      rke2-metrics-server:
+        repo: rancher-rke2-charts
+        version: 3.12.002
+      rke2-ingress-nginx:
+        repo: rancher-rke2-charts
+        version: 4.9.100
```

Comment:
I think the versions exactly match what is in v1.28.8, so we can just pull from that and not update anything. Ref: https://github.com/rancher/rke2/releases/tag/v1.29.3%2Brke2r1